tests/test_ide_integration.py

```diff
@@ -0,0 +1,112 @@
+#    Copyright 2018 D-Wave Systems Inc.
+
+#    Licensed under the Apache License, Version 2.0 (the "License")
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+
+#        http: // www.apache.org/licenses/LICENSE-2.0
+
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import unittest
+# import pytest
+import logging
+import os
+from subprocess import Popen, PIPE,STDOUT
+import inspect
+import time
+
+# These are integration tests with leap ide
+# This require loding leap ide docker image and run inside that image
+class LeapIdeIntegrationTests(unittest.TestCase):
+
+    def setUp(self):
+        self.verificationErrors = []
+        self.logger = None
+        self.log = self.custom_logger(logging.DEBUG)
+
+    def custom_logger(self,log_level=logging.DEBUG):
```

Comment:
I am probably out of my depth here, but it seems like overkill to have a custom logger for such a small test.

I believe this test code is meant to be pulled into Jenkins and tested with a specific docker image. As such, is it possible for the custom logger code to live there rather than directly in the test code?

Reply:
to be consistent with standards followed here , will remove the custom logger from here and find other way to have logs